lib/rules/no-route-action.js

```diff
@@ -1,15 +1,16 @@
 import Rule from './_base.js';
 
 const ERROR_MESSAGE = 'Do not use `route-action` as %. Instead, use controller actions.';
```

Comment:
Can we change this into a function? That's how we typically generate dynamic error messages instead of trying to do replacements. 

```
function makeErrorMessage(actionName) { ... }
```